VBLOCKS-1805 | Fix dtmf tones overrides (#176)
twilio/twilio-voice.js: CHANGELOG.md

2.6.1 (In Progress)
=====================

Bug Fixes
---------

> - Fixed an issue where custom dtmf sounds are not playing. With this release, custom dtmf sounds should now play when provided during device initialization.

Reviewer: ```suggestion
- Fixed an issue where custom DTMF sounds would not play. With this release, custom DTMF sounds should now play when configured during device initialization.
```
